### ISSUE TITLE
v2.6.0 The first instance cannot keep the window on top

### ISSUE THREAD

**Issue by aaaa7412** [Arturo-1212/PTCGPB#7]
Other instances can display the top window normally.

![螢幕擷取畫面 2024-12-25 121954](https://github.com/user-attachments/assets/3822c50e-fb0d-4a5f-9753-f5c2046a3b57)

**Comment by Arturo-1212** [comment]
Try the new version. This shouldn't be happening now

**Comment by aaaa7412** [comment]
I have switched to V2.7.0 and have not found the problem of keeping the window on top.

When I hung up using V2.6.0 all day today, I found a God pack, but there seemed to be a problem with the file size after automatic screenshots, resulting in the image file being unable to be opened.
Not sure if anyone else has this problem? Or is it just me?

The confirmed screenshot file size is 62 bytes (the picture is in Chinese)
![螢幕擷取畫面 2024-12-25 195105](https://github.com/user-attachments/assets/76b63c55-a8c9-49c7-a2ab-ac0f7382fd30)


In addition, I hope to add support for the Chinese version. Thank you Arturo for your hard work.
